pkg/scheduler/framework/session_plugins.go

```diff
@@ -173,9 +173,9 @@ func (ssn *Session) Reclaimable(reclaimer *api.TaskInfo, reclaimees []*api.TaskI
 				victims = nil
 				break
 			}
-			if !init {
+			if init {
 				victims = candidates
-				init = true
+				init = false
```

Comment:
My another question is: can we not need this init variable? If it is triggered for the first time, len(victims) should be nil, can we use this directly to judge?
```go
                               if abstain == 0 {
				continue
			}
			if len(candidates) == 0 {
				victims = nil
				break
			}
			if victims == nil {
				victims = candidates
			} else {
				var intersection []*api.TaskInfo
				// Get intersection of victims and candidates.
				for _, v := range victims {
					for _, c := range candidates {
						if v.UID == c.UID {
							intersection = append(intersection, v)
						}
					}
				}

				// Update victims to intersection
				victims = intersection
			}

```

Reply:
This approach is even better - but proper comment can me useful for a reader.